Iso_Level: 0.5
GR_x: 32
GR_y: 32
GR_z: 32